sklearn/linear_model/coordinate_descent.py

```diff
@@ -1612,13 +1612,13 @@ class MultiTaskElasticNet(Lasso):
 
     The optimization objective for MultiTaskElasticNet is::
 
-        (1 / (2 * n_samples)) * ||Y - XW||^Fro_2
+        (1 / (2 * n_samples)) * ||Y - XW||_Fro^2
         + alpha * l1_ratio * ||W||_21
         + 0.5 * alpha * (1 - l1_ratio) * ||W||_Fro^2
 
     Where::
 
-        ||W||_21 = \\sum_i \\sqrt{\\sum_j w_{ij}^2}
+        ||W||_21 = sum_i sqrt(sum_j w_ij ^ 2)
```

Comment:
Maybe -1 for this one. It seems not friendly to latex users. More importantly, it's not consistent with other parts of the doc (e.g., MultiTaskLasso).

Reply:
The comments above suggest that docstrings should not contain any latex.

Reply:
I won't be too strict on these minor issues.